src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/RedisArraySessionHandlerTest.php

```diff
@@ -13,7 +13,10 @@
 
 class RedisArraySessionHandlerTest extends AbstractRedisSessionHandlerTestCase
 {
-    protected function createRedisClient(string $host): \RedisArray
+    /**
+     * @return \RedisArray|object
```

Comment:
this change prevents PHP from failing with `because class RedisArray is not available` when the extension is not loaded